muon/domain/messages.js

```diff
@@ -110,7 +108,7 @@ exports.failure = function(protocol, status, text) {
     return msg;
 }
 
-exports.muonMessage = function(payload, sourceService, remoteServiceUrl) {
+exports.muonMessage = function(payload, sourceService, remoteServiceUrl, step) {
```

Comment:
the step name matters. the java side switches on this to figure out whats happened and whether to throw an error